content/docs/reference/config/installation-options-changes/index.md

```diff
@@ -17,443 +17,237 @@ The tables below show changes made to the installation options used to customize
-| `global.proxy.envoyStatsd.host` | `istio-statsd-prom-bridge` | `` |  | `example: statsd-svc.istio-system` |
-| `global.proxy.envoyStatsd.port` | `9125` | `` |  | `example: 9125` |
-| `global.proxy_init.image` | `proxy_init` | `proxy_init` |  | `Base name for the proxy_init container, used to configure iptables.` |
-| `global.controlPlaneSecurityEnabled` | `false` | `false` |  | `controlPlaneMtls enabled. Will result in delays starting the pods while secrets arepropagated, not recommended for tests.` |
-| `global.disablePolicyChecks` | `false` | `true` |  | `disablePolicyChecks disables mixer policy checks.if mixer.policy.enabled==true then disablePolicyChecks has affect.Will set the value with same name in istio config map - pilot needs to be restarted to take effect.` |
-| `global.enableTracing` | `true` | `true` |  | `EnableTracing sets the value with same name in istio config map, requires pilot restart to take effect.` |
-| `global.mtls.enabled` | `false` | `false` |  | `Default setting for service-to-service mtls. Can be set explicitly usingdestination rules or service annotations.` |
-| `global.oneNamespace` | `false` | `false` |  | `Whether to restrict the applications namespace the controller manages;If not set, controller watches all namespaces` |
-| `global.configValidation` | `true` | `true` |  | `Whether to perform server-side validation of configuration.` |
-
-### Modified `gateways` key/value pairs
-
-| Key | Old Default Value | New Default Value | Old Description | New Description |
-| --- | --- | --- | --- | --- |
-| `gateways.istio-ingressgateway.type` | `LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be` | `LoadBalancer` |  | `change to NodePort, ClusterIP or LoadBalancer if need be` |
-| `gateways.istio-egressgateway.enabled` | `true` | `false` |  |  |
-| `gateways.istio-egressgateway.type` | `ClusterIP #change to NodePort or LoadBalancer if need be` | `ClusterIP` |  | `change to NodePort or LoadBalancer if need be` |
-
-### Modified `certmanager` key/value pairs
-
-| Key | Old Default Value | New Default Value | Old Description | New Description |
-| --- | --- | --- | --- | --- |
-| `certmanager.tag` | `v0.3.1` | `v0.6.2` |  |  |
-
-### Modified `kiali` key/value pairs
-
-| Key | Old Default Value | New Default Value | Old Description | New Description |
-| --- | --- | --- | --- | --- |
-| `kiali.tag` | `istio-release-1.0` | `v0.14` |  |  |
+| `global.tag` | `release-1.1-latest-daily` | `master-latest-daily` | `Default tag for Istio images.`  | `Default tag for Istio images.` |
+| `global.proxy.resources.limits.memory` | `128Mi` | `1024Mi` |  |  |
+| `global.proxy.dnsRefreshRate` | `5s` | `300s` | `Configure the DNS refresh rate for Envoy cluster of type STRICT_DNS 5 seconds is the default refresh rate used by Envoy`  | `Configure the DNS refresh rate for Envoy cluster of type STRICT_DNS This must be given it terms of seconds. For example, 300s is valid but 5m is invalid.` |
```

Comment:
@howardjohn do you recall why this change is made?

Reply:
Yes. 5s means that for every DNS ServiceEntry you make (+2 for pilot and tracing), a DNS request will be made, for each proxy. And due to how kube-dns works generally a dns request actually becomes 6 requests for external urls. So people were having their DNS overloaded.

https://github.com/istio/istio/issues/13710 for details

No one should need to configure it now that 5m is the default, and we will replace it with proper TTL support soon

Reply:
Thanks John, sounds like this is something user can ignore during upgrade (e.g. have a better default in 1.2)